data-table/table.js

```diff
@@ -361,18 +368,104 @@ function createEntryCell(config, row, rowIndex, colIndex) {
                 option.innerHTML = config.datumConfig.values[fieldNum][i];
                 select.appendChild(option);
             }
-            label.appendChild(select);
+            field = select;
         } else {
             /**
              * FIXME: This error handling could be improved. Maybe a try-catch block?
              */
-            let errorText = "Cell field datatype not supported.";
-            throw errorText;
+            throw String("Cell field datatype not supported.");
         }
+
+        if (cellFieldHasCallback(config, fieldNum)) {
+            createCallbackListener(config, cell, field, fieldNum)
+        }
+
+        label.appendChild(field);
         cell.appendChild(label);
     }
 }
 
+/**
+ * Checks if a given field has a callback associated with it
+ * @param {object} config   - Table configuration object
+ * @param {Number} fieldNum - Index of the field within the cell
+ * @returns {boolean}       - Returns True if field has a callback
+ */
+function cellFieldHasCallback(config, fieldNum) {
+    return config.datumConfig.callbacks[fieldNum] !== undefined && config.datumConfig.callbacks[fieldNum] !== "None";
```

Comment:
A magic string `None` seems scary - can you use `null` instead (here and on line 70)?

Reply:
Done.